snippets/application.md

```diff
@@ -11,7 +10,7 @@ a notebook listing the available dataset tables is [here](https://github.com/ope
 the latest dataset import dates is described [here](https://github.com/opensafely/database-notebooks/blob/master/notebooks/database-builds.ipynb).
 Further information regarding the platform can be found [here](https://docs.opensafely.org/).
 
-**All outputs from the OpenSAFELY-EMIS/-TPP NHS England COVID-19 research platform must be aggregated data with small number suppression applied.
+**All outputs from the NHS England OpenSAFELY COVID-19 servce must be aggregated data with small number suppression applied.
```

Comment:
```suggestion
**All outputs from the NHS England OpenSAFELY COVID-19 service must be aggregated data with small number suppression applied.
```

Reply:
Thanks Iain for picking up on this error. Much appreciated.